backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/LookupController.cs

```diff
@@ -65,7 +66,6 @@ public async Task<ActionResult<Party>> GetOrganisation(string orgNummer)
         /// <param name="partyId">The partyId for the reportee to look up</param>
         /// <returns>Reportee if party is in authenticated users reporteelist</returns>
         [HttpGet]
-        [Authorize]
```

Comment:
This still requires [Authorize]

Reply:
Woops, this file snuck in along with the other changes I did in the other files when I tested why LookupController failed. Reverted this file and pushed so nothing here is changed now